content/applications/websites/livechat/overview/get_started.rst

```diff
@@ -2,82 +2,288 @@
-=======================
+Before participating in a live chat, operators should update their :guilabel:`Online Chat Name`.
+This is the name that will be displayed to site visitors in the live chat conversation.
 
-Go to :menuselection:`Website --> Configuration --> Settings`, access your channel under
-*Live Chat*, and *Join Channel* or *Leave Channel*.
+To update the :guilabel:`Online Chat Name`, click on the user name in the upper-right corner of any
+page in the database. Select :guilabel:`My Profile` to open the :guilabel:`Profile` page. On the
+right side of the :guilabel:`Preferences` tab, locate the :guilabel:`Online Chat Name` field and
+enter the preferred name.
 
-.. image:: get_started/joinchannel.png
+.. image:: get_started/live-chat-my-profile.png
    :align: center
-   :alt: View of a channel form and the option to join a channel for Odoo Live Chat
+   :alt: View of the My Profile option in Odoo.
+
+If an :guilabel:`Online Chat Name` is not set, the name displayed will default to the
+:guilabel:`User Name`.
+
+.. example::
+   A user has their full name as their :guilabel:`User Name`, but they do not want to include their
+   last name when participating in a live chat conversation. They would then set their
+   :guilabel:`Online Chat Name` to include only their first name.
+
+  .. image:: get_started/live-chat-online-chat-name.png
+     :align: center
+     :alt: View of user profile in Odoo, emphasizing the Online Chat name field.
+
+Join or leave a channel
+-----------------------
+
+To join a live chat channel, go to the :menuselection:`Live Chat` app and click the
+:guilabel:`JOIN` button on the kanban card for the appropriate channel.
+
+Any channel where the user is currently active will show a :guilabel:`LEAVE` button. Click this
+button to disconnect from the channel.
+
+.. image:: get_started/live-chat-leave-channel.png
+   :align: center
+   :alt: View of a channel form and the option to join a channel for Odoo Live Chat.
+
+.. important::
+   *Operators* that do not show any activity in Odoo for more than thirty minutes will be
+   considered disconnected, and subsequently removed from the channel.
+
+Manage live chat requests
+-------------------------
+
+When an operator is active in a channel, chat windows will open in the bottom right corner of the
+screen, no matter where they are in the database. They can participate in conversations without
+leaving their current location.
+
+.. tip::
+   Conversations can also be accessed by clicking the :guilabel:`Conversations` icon in the
+   :guilabel:`Menu Bar`.
+
+   .. image:: get_started/live-chat-menu-bar.png
+      :align: center
+      :alt: View of the menu bar in Odoo emphasizing the conversations icon.
+
+Live chat conversations can also be viewed by navigating to :menuselection:`Dashboard --> Discuss`.
+New conversations will appear in bold under the :guilabel:`LIVECHAT` heading along the left panel.
+
+.. image:: get_started/live-chat-managing-chat-responses.png
```

Comment:
all images - we dont need to prepend `live-chat-` to each image filename since it is already implied by residing within the folder